CHANGELOG.rst

```diff
@@ -1,6 +1,30 @@
 Change log
 ==========
 
+0.19.1 (2020-11-06)
+-------------------
+
+New features
+~~~~~~~~~~~~
+
+* **config:** add classes :class:`~universum.configuration_support.Step` and
```

Comment:
add :class:`~universum.configuration_support.Step` and :class:`~universum.configuration_support.Configuration` classes